handlers/get_network_handler.py

```diff
@@ -0,0 +1,25 @@
+import json
+from typing import Union
+
+from aws_lambda_context import LambdaContext
+
+from usecases.get_network import GetNetwork
+from utils.wrappers.aws.api_gateway import ApiGateway, ApiGatewayEvent
+
+
+@ApiGateway()
+def handle(event: ApiGatewayEvent, __: LambdaContext, get_network: Union[GetNetwork, None] = None) -> dict:
+
+    get_network = get_network or GetNetwork()
+    response = get_network.get(event.path['hash'])
+
+    if response is None:
+        raise Exception('not_found')
```

Comment:
Shoudn't we improve this Exception messasge?

Reply:
Maybe return 404?

Reply:
The `404` is already returned by the wrapper: https://github.com/HathorNetwork/hathor-explorer-service/pull/3/files/cb3a9848499dcc99b381ac9e3ae749c9b68634ca#diff-d28d8d9a7df84a0eb08eb60fcdc5dd0a7567d533e7f95a2281993d34b3a1fabdR36

I agree that we can improve the returned message, but the exception in this case should match the dict key defined in the wrapper.

I've just created an issue for that: https://github.com/HathorNetwork/hathor-explorer-service/issues/8